packages/dev/core/src/Engines/abstractEngine.ts

```diff
@@ -919,21 +920,30 @@ export abstract class AbstractEngine {
             return;
         }
 
-        this._minFrameTime = 1000 / (value + 1); // We need to provide a bit of leeway to ensure we don't go under because of vbl sync
+        this._minFrameTime = 1000 / value;

```

Comment:
Leaving a comment here to prevent merging before this is resolved.
Need to understand why it was `value+1` before.

Reply:
After the accumulator pattern, it’s not needed. I think it was added as a buffer to ensure it doesn’t exceed the target frame. And yes, let’s wait for someone from the team to approve it.

Reply:
I am from the team (👋 ), so... approved :-)

Reply:
Yeah I know 👋🏻 I thought you wanted to get verification (before merge) from someone from the team besides you 🙏🏻

Reply:
I do want @deltakosh to comment here. It will probably take some time. I'll move it to draft in the meantime.